doc/man/man3/crypto_verify16.3monocypher

```diff
@@ -17,16 +17,15 @@
 .Sh DESCRIPTION
 Cryptographic operations often require comparison of secrets or values
 derived from secrets.
-Standard comparison functions tend to exit as soo as they find a
-difference, and doing so leaks information through variable timings.
+Standard comparison functions (like memcmp) tend to exit when they find
```

Comment:
Maybe we should remove the parentheses here?

Reply:
agreed, matches the "like memset and bzero" in the crypto_wipe PR